docs/reference/setup/install/zip-windows.asciidoc

```diff
@@ -90,9 +90,9 @@ such as `node.name` could be specified on the command line.
 include::check-running.asciidoc[]
 
 [[windows-service]]
-==== Installing Elasticsearch as a Service on Windows
+==== Install {es} as a Service on Windows
```

Comment:
We should add some sentences on what happens with security when elasticsearch is run as a service. There will be no tty attached when elasticsearch starts and we won't print out credentials and tokens. We should 
1. Mention that after ES is started, users can run the `elasticsearch-reset-password -u elastic` CLI to get/set the password for the elastic 
2. Mention that they can run `elasticsearch-create-enrollment-token` for getting a token for kibana instances 

Happy to work through this in a windows VM and push a commit with suggestions

Reply:
Auto-confiuration doesn't kick in at all when windows runs as a service because it doesn't use the `elasticsearch.bat` file to start elasticsearch. This is a miss , I'll open an issue an see if we can address this. For the context of this PR,  I think we should clarify that when Elasticsearch is installed as a service: 
- Authentication is enabled and users can get the  password for the elastic user with `elasticsearch-reset-password -u elastic`
- TLS is not enabled/configured.

When we address the functionality gap, we can update this doc to mention the enrollment process.

Reply:
I just started a Windows Server VM and downloaded the latest 8.0.0-beta1. I ran `elasticsearch.bat`, which printed the generated password for the `elastic` user to the command line. It went by quickly and I had to scroll back pretty far to find it, but it was there!

![image](https://user-images.githubusercontent.com/25848033/141165012-4692adb5-06d1-4507-aa50-b9130925807a.png)

Reply:
This is not running elasticsearch as a service, see the relevant section in the doc for how to install and run es as a service

Reply:
Is this not the correct section? https://www.elastic.co/guide/en/elasticsearch/reference/8.0/zip-windows.html#windows-service

Do you have to run `elasticsearch-service.bat install [service-id]` and then `elasticsearch-service.bat start`?

Reply:
FWIW, when I run `elasticsearch-service.bat install`, I get this error:

```
ERROR: Positional arguments not allowed, found [install]
```

Reply:
> Is this not the correct section? https://www.elastic.co/guide/en/elasticsearch/reference/8.0/zip-windows.html#windows-service
> 
> Do you have to run `elasticsearch-service.bat install [service-id]` and then `elasticsearch-service.bat start`?

Yes this is the correct section and these are the comands you need to run . Your original reproduction didn't

> I ran elasticsearch.bat, 

and this is what I was pointing out

Reply:
> FWIW, when I run `elasticsearch-service.bat install`, I get this error:
> 
> ```
> ERROR: Positional arguments not allowed, found [install]
> ```

Nor sure, didn't happen to my when I tried that

Reply:
User error is the culprit. I was running  `elasticsearch.bat` instead of `elasticsearch-service.bat`.

Reply:
I reworked this section a bit in the latest push, and made a distinction between installing/running and managing as a service on Windows.